include/png_output.h

```diff
@@ -0,0 +1,54 @@
+#include <png.h>
```

Comment:
If we have a file just for png output, we should make something similar for ppm and jpg, and abstract these out to some image class, and output.h should only call on these classes